plugin/inc/installsteps/handleprofilerightsinstallstep.class.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace GlpiPlugin\Iservice\InstallSteps;
+
+class HandleProfileRightsInstallStep
+{
+    const CLEANUP_ON_UNINSTALL = true;
+
+    public static function do(): bool
+    {
+        global $DB;
+
+        $table = 'glpi_profilerights';
+        $superAdminProfileId = \Profile::getSuperAdminProfilesId();
+        $rightsValues = include __DIR__ . DIRECTORY_SEPARATOR . '..' . DIRECTORY_SEPARATOR . 'rights.config.php';
+        $result = false;
+
+        if (!self::backupTable($table)) {
+            return false;
+        }
+
+        foreach ($rightsValues as $rightName => $rightValue) {
+            $result = $DB->update(
+                $table,
+                [
+                    'rights' => $rightValue,
+                ],
+                [
+                    'WHERE'  => [
+                        'name' => $rightName,
+                        'profiles_id' => ['!=', $superAdminProfileId],
+                    ],
+                ]
+            );
+
+            if ($result === false) {
+                return false;
+            }
```

Comment:
You can totally get rid of the `$result` variable
```suggestion
            if (!$DB->update(
                $table,
                [
                    'rights' => $rightValue,
                ],
                [
                    'WHERE'  => [
                        'name' => $rightName,
                        'profiles_id' => ['!=', $superAdminProfileId],
                    ],
                ]
            )) {
                return false;
            };
```